rmw_connext_cpp/bin/apply-patch.py

```diff
@@ -79,7 +80,13 @@ def apply_patch(s, patch):
     try:
         content_out = apply_patch(content_in, content_patch)
     except Exception:
-        print(i, p, o)
+        print(i, p, o, file=sys.stderr)
```

Comment:
How about making this a "full message" like: "Failed to generate '{o}' using input '{i}' and patch '{p}"?

Reply:
done